packages/organizations/origin-organization-irec-api/src/beneficiary/beneficiary.entity.ts

```diff
@@ -11,9 +11,22 @@ export class Beneficiary extends ExtendedBaseEntity {
     irecBeneficiaryId: number;
 
     @Column()
-    organizationId: number;
+    @IsOptional()
+    organizationId?: number;
```

Comment:
Does the database has NOT NULL constaint or not?

Reply:
I've added
```
ALTER TABLE "irec_beneficiary" ALTER COLUMN "organizationId" DROP NOT NULL
```
In mirgation section